test/test_utils/sagemaker.py

```diff
@@ -326,10 +339,11 @@ def execute_local_tests(image, pytest_cache_params):
             image,
             ec2_ami_id,
             ec2_key_name,
-            region
+            region,
         )
         ec2_conn = ec2_utils.get_ec2_fabric_connection(instance_id, key_file, region)
         ec2_conn.put(sm_tests_tar_name, f"{UBUNTU_HOME_DIR}")
+        install_python_in_instance(ec2_conn, python_version="3.9")
```

Comment:
Remind me, what was the reasoning behind 3.9?

Reply:
Python 3.10 doesn't support a number of PyTorch Ecosystem packages yet. Python 3.9 is as high as I could go.